org.kde.calligra.json

```diff
@@ -0,0 +1,342 @@
+{
+    "id": "org.kde.calligra",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "calligra",
+    "finish-args": ["--share=network", "--share=ipc", "--socket=x11", "--socket=wayland" ],
```

Comment:
it's `wayland` + `fallback-x11` or just `x11` not both.

Also please format this in a readable way.